### PR TITLE
Fix Supplier ID in Test JSON

### PR DESCRIPTION
**Pull request by tgeeting**
Supplier ID needs to be updated to 20 in all TripleLift related Test JSON files.

https://travis-ci.org/prebid/prebid-server/jobs/601297448?utm_medium=notification&utm_source=github_status
(Travie made a boo boo).

### CODE REVIEW

File: adapters/triplelift/triplelifttest/exemplary/simple-video.json [hunk 1]
```diff
@@ -1,114 +1,144 @@
 {
-	"mockBidRequest": {
```
Comment (Kevin-P-Kerr) [inline adapters/triplelift/triplelifttest/exemplary/simple-video.json:2]: why is this test in particular so different?

Comment (tgeeting) [reply adapters/triplelift/triplelifttest/exemplary/simple-video.json:2]: It's not, VSCode just auto-formatted the JSON indents to 2 spaces 🤷‍♂

Comment (Kevin-P-Kerr) [reply adapters/triplelift/triplelifttest/exemplary/simple-video.json:2]: can you ./validate.sh

Comment (Kevin-P-Kerr) [reply adapters/triplelift/triplelifttest/exemplary/simple-video.json:2]: that will format the ws in the standard way

**Review by Kevin-P-Kerr — APPROVED** [review]

**Pull request merged**